content/applications/general/users/delete_account.rst

```diff
@@ -0,0 +1,35 @@
+===========================
+Odoo.com Account Management
+===========================
+
+Account management
+==================
+
+Overview
+--------
+
+To delete your Odoo account, access the delete option by clicking on the user icon. The delete
+option can be accessed by going to :menuselection:`My Account --> Edit Security Settings -->
+Delete Account`. It can also be accessed by going to https://www.odoo.com/my/home.
+
+Deletion Process
+----------------
+
+Click on the user icon and select :guilabel:`My Account`. Then click on :guilabel:`Edit Security
+Settings`.
+
+Next, under the :guilabel:`Delete Account` section, click on the :guilabel:`Delete Account` button.
```

Comment:
Assuming this doc is *only* about account deletion (and we're not planning to revise the overview later to make room for new content), then either the Overview or the first sentence should be revised since it's redundant.

Suggestion below:

```suggestion
On the :guilabel:`Security Settings` page, scroll to the :guilabel:`Delete Account` section, and
then click on the :guilabel:`Delete Account` button.
```

Reply:
Noted, I've reduced the number of sentences. Thanks.